battleship-client/src/test/java/com/academy/solid/nie/client/communication/SocketClientTest.java

```diff
@@ -39,7 +39,6 @@ public void should_println_readLine_whenCallRunOnClient() throws IOException {
         socketClient.send(message);
         //then
         verify(printWriter).println(message);
-        verify(bufferedReader).readLine();
     }
 
     @Test(groups = {"unit"})
```

Comment:
I am not sure it would work for you but in our project we only explicitly declare the integration group, and unit is just the default one